examples/basic/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "tsc",
+    "build": "armkit generate && tsc",
```

Comment:
Rather having a seperate cmd for this : 
`"generate": "armkit generate"`

Reply:
Yes, I left that one in too. But in any case, the example cannot be built with `tsc` unless `generate` has been run before, so I thought it would make sense to include it directly in the build command as well.

Reply:
there are a ot of occasions where you want to run `build` w/o generating the whole lib (once this is complete it is a significant compile time)

Reply:
Yeah, makes sense. Let me try something else :-)

Reply:
Just pushed another proposal: only run `tsc` if the `.generated` directory is present. If not, do not attempt to build (it will fail anyway). Then the user can go to the example dir, and run `yarn generate` and `yarn build`.